products/apache-http-server.md

```diff
@@ -7,14 +7,14 @@ iconSlug: apache
 layout: post
 link: https://httpd.apache.org/
 category: server-app
-activeSupportColumn: true
+activeSupportColumn: false
 command: httpd -v
-releaseColumn: false
+releaseColumn: true
 releaseDateColumn: true
 sortReleasesBy: "releaseCycle"
 releases:
   - releaseCycle: "2.4"
-    release: 2012-02-21
+    release: 2021-12-20
```

Comment:
nope , we put release dates of related branch's .0 version , so this date should be 2012 not 2021 because 2.4.0 released in that time :)

Reply:
Kk, makes sense. Where does one put the release date which corresponds to the most recent patch release?

Reply:
kk, updated